src/clojure_lsp/main.clj

```diff
@@ -3,6 +3,7 @@
     [clojure-lsp.db :as db]
     [clojure-lsp.handlers :as handlers]
     [clojure-lsp.interop :as interop]
+    [clojure-lsp.parser :as parser]
```

Comment:
Could you please move `document->decoded-uri` out of parser and into `clojure-lsp.interop` where I think these kind of client fudges are more appropriate.